src/transactions/feed/TransferFeedItem.tsx

```diff
@@ -51,6 +50,7 @@ function TransferFeedItem({ transfer }: Props) {
           recipient={recipient}
           status={transfer.status}
           transactionType={transfer.__typename}
+          transactionSubtype={transfer.type}
```

Comment:
nit: perhaps it'd be less confusing if we renamed these props to `transactionTypename` and `transactionType`?